src/breaking-changes/summary.md

```diff
@@ -4,5 +4,30 @@ Breaking changes should be avoided when possible.
 [RFC 1105](https://rust-lang.github.io/rfcs/1105-api-evolution.html) lays the foundations for what constitutes a breaking change.
 Breakage may be deemed acceptable or not based on its actual impact,
 which can be approximated with a [crater](https://github.com/rust-lang/crater/blob/master/docs/bot-usage.md) run.
+Running crater should be done if nontrivial breakage is expected, so the information is
+available during the final comment period.
 
-If the impact isn't too high, looping in maintainers of broken crates and submitting PRs to fix them can be a valid strategy.
+If the impact isn't too high, looping in maintainers of to-be-broken crates and submitting PRs
+to fix them (and ensuring those PRs have been merged and released) can be a valid strategy. However, this can only affect the crates in question, and
+it does not automatically affect their dependents. Binary dependents may have already locked-in
+a different, older version.
+
+## Breaking and the trains
+If a PR is merged and it turns out to have caused code to not compile during the nightly or beta release cycle,
+unless there is a trivial fix, the PR should be reverted and a crater run should assess the impact.
+
+### Model: A Trivial Fix
```

Comment:
AFAICT, this example is specifically about a nightly-only feature. It seems like it'd be a better example if we used a change that was visible to stable Rust code but that we mitigated, and how we mitigated it?

(Not trying to add substantial additional work here. And this example may still make sense as a guide for changes that only affect nightly.)

Reply:
@joshtriplett Actually the issue here is precisely that the change was visible to stable Rust code, i.e. the regression was heading to stable, but the feature was nightly. I can clarify that in the remarks if you like?

Reply:
Ah, I see. That does seem different from the case of a change where the change itself intentionally adds new stable API surface area, but I appreciate the clarification.

Reply:
Uh, so, I don't actually know what I'm supposed to do here?